openedx/core/djangoapps/util/row_delete.py

```diff
@@ -32,26 +32,27 @@ def delete_rows(
 
     # The "as id" below fools Django raw query into thinking the primary key is being queried.
```

Comment:
What are the implications of fooling Django in this way?

Reply:
It's necessary because Django will throw an exception if the raw SQL does not query the primary key. I'll add a comment stating that.